string label
int32 id
bool reliable
bool ordered
string protocol
int32 chunk_size
string state